js/form.js

```diff
@@ -13,9 +15,19 @@ const scaleControl = imgForm.querySelector('.scale__control--value');
 const effectLevelValue = imgForm.querySelector('.effect-level__value');
 const effectDefault = imgForm.querySelectorAll('#effect-none');
```

Comment:
```suggestion
const effectDefault = imgForm.querySelector('#effect-none');
```

Есть только один такой элемент, поэтому нужен метод `querySelector`